spine_json_lib/data/data_types/draworder.py

```diff
@@ -71,11 +71,17 @@ def _adjust_draw_oder_offset_with_erased_slots(
         adjusted_draw_order_offsets = copy.deepcopy(slot_offsets)
 
         for name, slot_offset in adjusted_draw_order_offsets.items():
+            if name in slots_to_be_removed:
+                continue
             index_original = clean_original_slots.index(name)
             index_final = clean_reordered_slots.index(name)
             slot_offset.offset = index_final - index_original
 
-        return adjusted_draw_order_offsets
+        return [
+            offset
+            for offset in adjusted_draw_order_offsets.values()
+            if offset.slot not in slots_to_be_removed
+        ]
```

Comment:
Will the offsets stay correct if we update them counting slots that are going to be removed?

Indexs will change once we remove the slots.

Reply:
Final indexes are already ignoring the removed slots as you can see in `clean_original_slots` and `clean_reordered_slots` where slots_to_be_removed are already filtered out, so it won't be a problem.